.github/workflows/dmd-cxx.yml

```diff
@@ -0,0 +1,99 @@
+# Github action to test C++ implementation of DMD
+#
+# Based-on:
+#  - Github C/C++ with Make
+#  - DMD C++ interop tests
+name: DMD C++
+
+on:
+  pull_request:
+    branches: [ dmd-cxx ]
+  push:
+    branches: [ dmd-cxx ]
+
+jobs:
+  main:
+    name: Run
+    strategy:
+      matrix:
+        # Ubuntu 16.04 in order to support older LLVM / GCC
+        os: [ ubuntu-16.04 ]
+
+        target: [ clang, g++ ]
+
+        # This sets the configuration for each jobs
+        # There's a bit of duplication involved (e.g. breaking down g++-9.3 into 2 strings),
+        # but some items are unique (e.g. clang-9.0.0 and 4.0.1 have differences in their naming).
+        include:
+          # Clang boilerplate
+          - { target: clang, compiler: clang }
+          # g++ boilerplace
+          - { target: g++, compiler: g++ }
```

Comment:
Doesn't look like this is needed ?

Reply:
> Why the DMD master build ?

To test the bootstrap process is working.

Reply:
Maybe?  I guess can use `${{ matrix.target }}` instead of `matrix.compiler`.